ddtrace/__init__.py

```diff
@@ -11,7 +11,7 @@
     __version__ = pkg_resources.get_distribution(__name__).version
 except pkg_resources.DistributionNotFound:
     # package is not installed
-    __version__ = None
+    __version__ = 'not_installed'
```

Comment:
Why this change?

Reply:
Setting `__version__` to `None` results in an error when it attempts to set headers, since the value of a header can't be None.

Reply:
how are you using it? this hasn't been an issue for any of us before.

can you remove from this PR, if we make this change it should be in it's own PR

Reply:
I cloned the repo and symlinked `ddtrace` into a virtualenv for testing. I'll remove the change.

Reply:
ah, should do `python setup.py develop` for that